django/db/models/sql/query.py

```diff
@@ -800,8 +800,8 @@ def change_aliases(self, change_map):
         # "group by" and "where".
         self.where.relabel_aliases(change_map)
         if isinstance(self.group_by, tuple):
-            self.group_by = tuple([col.relabeled_clone(change_map) for col in self.group_by])
-        self.select = tuple([col.relabeled_clone(change_map) for col in self.select])
+            self.group_by = tuple(col.relabeled_clone(change_map) for col in self.group_by)
```

Comment:
Apparently using a list comprehension performs better here, see https://github.com/django/django/pull/7730/files#r93793194. If someone can explain when exactly that's the case, we can add it to the coding style guidelines in https://github.com/django/django/pull/8284.